app/models/courseInstructors.py

```diff
@@ -0,0 +1,7 @@
+from app.models import*
+from app.models.course import Course
+from app.models.user import User
+
+class CourseInstructors(baseModel):

```

Comment:
Follow the naming convention of the other tables. In general, they shouldn't be plural. The table name describes what each record in the table is: in this case, a single instructor.

Reply:
It is changed to single instructore now